src/JSystem/JStage/JSGSystem.cpp

```diff
@@ -0,0 +1,37 @@
+#include "JSystem/JStage/JSGSystem.hpp"
+#include "dolphin/types.h"
+
+JStage::TSystem::~TSystem() 
+{
+
+}
+
+s32 JStage::TSystem::JSGFGetType() const 
+{
+    return 1;
+}
+
+void* JStage::TSystem::JSGFindObject(const char*, JStage::TEObject) const 
+{
+    return NULL;

```

Comment:
use nullptr, same as the rest of the codebase (and several other decomp projects)